test/buffer/worker/002_slice_upload_test.go

```diff
@@ -6,61 +6,85 @@ import (
 
 // test002SliceUpload imports 2 more records; 6 >= 5 (uploadCountThreshold) - upload is triggered.
 func (ts *testSuite) test002SliceUpload() {
+	ts.t.Logf("-------------------------")
+	ts.t.Logf("002 slice upload")
+	ts.t.Logf("-------------------------")
+
 	// Run imports immediately after the last check to prevent the check during imports.
 	ts.WaitForLogMessages(15*time.Second, `
-[worker-node-%s][service][conditions]INFO  checked "2" opened slices
+[worker-node-%s][bufferWorker][service][conditions]INFO checked "2" opened slices
 	`)
 
-	// Import records
+	// Import records 5-6
 	for i := 5; i <= 6; i++ {
 		ts.Import(i)
 	}
 
 	// Periodic condition checks have detected that the UPLOAD conditions for both slices/exports have been met.
-	ts.WaitForLogMessages(15*time.Second, `
-[worker-node-%d][service][conditions]INFO  closing slice "%s": count threshold met, received: 6 rows, threshold: 5 rows
-[worker-node-%d][service][conditions]INFO  closing slice "%s": count threshold met, received: 6 rows, threshold: 5 rows
-`)
+	ts.t.Logf("waiting for upload conditions check ...")
+	if ts.WaitForLogMessages(15*time.Second, `
+[worker-node-%d][bufferWorker][service][conditions]INFO closing slice "%s": count threshold met, received: 6 rows, threshold: 5 rows
+[worker-node-%d][bufferWorker][service][conditions]INFO closing slice "%s": count threshold met, received: 6 rows, threshold: 5 rows
+`) {
+		ts.t.Logf("upload conditions met")
```

Comment:
Pridal som priebezne logovanie toho, co sa v teste deje.
Spolu s komentarmi + etcd dumps to sluzi aj ako priklad toho ako to cele funguje.